services/email.js

```diff
@@ -0,0 +1,54 @@
+const { createTransport } = require('nodemailer');
+const config = require('../config');
+
+class EmailService {
+
+    constructor() {
```

Comment:
When instantiating a service, we should try not to import config and instead pass the config variables into the constructor so that we can make multiple different versions if needed. Can you remove the config import and add email transport to the constructor?

Reply:
I've updated the changes.